charts/server/values.yaml

```diff
@@ -128,9 +157,6 @@ ingress:
     - host: chart-example.local
       paths:
         - path: /
-          backend:
-            serviceName: chart-example.local
-            servicePort: *servicePort
```

Comment:
Why this change?

Reply:
This values has no effect here / are deadcode.
The needed values are hardcoded (like it is needed).

https://github.com/woodpecker-ci/helm/blob/3c4c246b89c4e4849b62f62dab2b8a4c36521be6/charts/server/templates/ingress.yaml#L34-L41

Reply:
Thanks, I never looked at them in detail and always overwrite them anyhow. I am fine with removing them.